test/testsupport/ItSpec.scala

```diff
@@ -76,10 +82,31 @@ class ItSpec
 
   override def beforeEach(): Unit = {
     super.beforeEach()
-    EssttpBackend.BarsVerifyStatusStub.statusUnlocked() // required by almost all tests
     webDriver.manage().deleteAllCookies()
   }
 
+  // suitable for most tests
+  def stubActionDefaults(): StubMapping = {
+    stubCommonActionsWith(
+      authAllEnrolments = Some(Set(TdAll.payeEnrolment)),
+      authCredentials   = Some(Credentials("authId-999", "GovernmentGateway")),
+      barsLockoutExpiry = None
+    )
+  }
+  // custom action setup
+  def stubCommonActionsWith(
```

Comment:
with this we may not need `stubActionDefaults` - that method is just using these default values anyway. One method it easier to maintain than two. In the one method world maybe this becomes something like `stubCommonActions` and for most cases you'd pass in nothing and use the defaults